helm/happa/values.yaml

```diff
@@ -9,6 +9,9 @@ groupID: 101
 
 Installation:
   V1:
+    Customer: giantswarm
```

Comment:
Why add these here instead of adding them to the CI default values?
https://github.com/giantswarm/happa/blob/master/helm/happa/ci/default-values.yaml

Reply:
Mistake, will fix that